Hunting Queries/BehaviorAnalytics/Anomalous Login to Devices.yaml

```diff
@@ -17,5 +17,5 @@ query: |
  | where DevicesInsights.IsLocalAdmin == true
  | where ActivityType == "LogOn"
  | where ActionType == "InteractiveLogon"
- | where ActivityInsights.FirstTimeUserLoggedOnToDevice == true
+ | where ActivityInsights contains "True"
```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.